docs/hardware/devices/rak/peripherals.mdx

```diff
@@ -16,6 +16,7 @@ values={[
 {label: 'User Button', value: 'User Button'},
 {label: 'GPS Module', value: 'GPS'},
 {label: 'Buzzer', value: 'Buzzer'},
+{label: 'Vibramotor', value: 'Vibramotor'},
```

Comment:
This tab will not work because it does not match the `value` of your `TabItem` lower in the file. I would recommend updating the value to match the title of the section. Might I recommend `Vibration` as the title?